tools/coqdep/lib/fl.ml

```diff
@@ -107,7 +107,12 @@ let findlib_resolve ~meta_files ~file ~package ~plugin_name =
     in
     List.fold_right add path file
   in
-  let meta_file = normalize_path (to_relative_path meta_file) in
+  let meta_file =
+    match Sys.getenv_opt "DUNE_SOURCEROOT" with
```

Comment:
Can we maybe add a comment to this?

Reply:
added a little comment

Reply:
Thanks, will look into this PR tomorrow, it's been a while so I need to refocus on this; but IIRC we should be able to get this merged as-is.

Reply:
Not sure if the normalization here is useful anymore. When called through gen_rules it will get fed into fixup_install_locations anyway, so fixup_install_locations could handle it.
Not sure about when it's called in a regular dune-coq project though.